lib/rucio/tests/test_bin_rucio.py

```diff
@@ -2327,3 +2331,22 @@ def test_admin_rse_update_unsupported_option(self):
         print(out, err)
         assert exitcode == 0
         assert not err
+
+    @pytest.mark.noparallel(reason='Modify config')
+    def test_lifetime_cli(self):
+        """ CLIENT(USER): Check CLI to declare lifetime exceptions """
+        # Setup data for CLI check
+        tmp_dsn_name = 'container' + rse_name_generator()
+        tmp_dsn_did = self.user + ':' + tmp_dsn_name
+        self.did_client.add_did(scope=self.user, name=tmp_dsn_name, did_type='DATASET')
+        self.did_client.set_metadata(scope=self.user, name=tmp_dsn_name, key='eol_at', value=(datetime.utcnow() - timedelta(seconds=86400)).strftime('%Y-%m-%d'))
+        tomorrow = datetime.utcnow() + timedelta(days=1)
+        tomorrow = tomorrow.strftime('%Y-%m-%d')
+        self.config_client.set_config_option(section='lifetime_model', option='cutoff_date', value=tomorrow)
```

Comment:
The first two lines should be merged. The main reason is that, especially now that we’re aiming for type hints, we should avoid changing the type of a variable wherever possible.

But then, I’m wondering whether it is actually necessary to assign this to a variable at all (you don’t do so above).